rag-and-text-to-sql/rag_code.py

```
@@ -0,0 +1,364 @@
+import json
+import re
+from pydantic_core.core_schema import AnySchema
```

Ruff (0.8.2)

`pydantic_core.core_schema.AnySchema` imported but unused

Remove unused import: `pydantic_core.core_schema.AnySchema`

(F401)